.kitchen.vagrant.yml

```diff
@@ -0,0 +1,101 @@
+---
+driver:
+  name: vagrant
```

Comment:
Could you add this driver to the `Gemfile`depdencies.

Reply:
Fixed